src/server/default/DefaultConfiguration.ts

```diff
@@ -0,0 +1,6 @@
+import { Configuration } from '../../types/Configuration';
+
+export const defaultConfiguration: Configuration = {
+    localAndroid: true,
```

Comment:
ERROR in /Users/$USER/ws-scrcpy/src/server/default/DefaultConfiguration.ts
    [tsl] ERROR in /Users/$USER/ws-scrcpy/src/server/default/DefaultConfiguration.ts(4,5)
          TS2322: Type '{ localAndroid: boolean; localIos: boolean; }' is not assignable to type 'Configuration'.
      Object literal may only specify known properties, and 'localAndroid' does not exist in type 'Configuration'.

Reply:
Thanks. This file is not used anymore.

Reply:
Dont see any error with removing this file. But dont see the connected devices list when localhost:8000 is opened.

Reply:
No device at all or just no ios-devices, or no devices from "remote tracker"?
How did you build and run it?

Reply:
no devices at all. 
build steps:
npm i
npm start
open localhost:8000 in browser

Reply:
You must use environment variables. Read this PR description. Something like this:
`INCLUDE_APPL=1 INCLUDE_GOOG=1 WS_QVH_PATH=/Users/myname/go/ws-qvh WS_SCRCPY_CONFIG=/Users/myname/ws-scrcpy/config.json npm start`

I know, this is not convenient.

Reply:
worked.